.travis.yml

```diff
@@ -34,6 +34,7 @@ before_install:
   - sudo /usr/bin/pip install coveralls
 install:
   - sudo /usr/bin/python setup.py install
+  - sudo /usr/bin/pip install mock
```

Comment:
Include in ``setup.py:setup(.., test_require={}, ..)``. See also, https://github.com/Connexions/cnx-publishing/blob/908221cce57e8c373d78acf51c437b1806317ec4/setup.py#L27